README.md

```diff
@@ -65,30 +53,49 @@ records the following results:
 $ export PATH=$GOROOT/bin:$GOPATH/bin:$PATH 
 ```  
 
-[gimme]: https://github.com/travis-ci/gimme
-
-### Download and run
+### Install
```

Comment:
Should probably list non-go-get installation instructions (just link to our releases/binaries). For some flows this is going to be the more common installation path.